gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/VoltageCheckService.java

```diff
@@ -53,13 +57,40 @@ public Optional<VoltageMonitoringResult> runVoltageCheck(SweData sweData, Dichot
             Crac crac = sweData.getCracFrEs().getCrac();
             Network network = getNetworkWithPra(dichotomyResult);
             VoltageMonitoring voltageMonitoring = new VoltageMonitoring(crac, network, dichotomyResult.getHighestValidStep().getRaoResult());
-            return Optional.of(voltageMonitoring.run(LoadFlow.find().getName(), LoadFlowParameters.load(), 4));
+            VoltageMonitoringResult result = voltageMonitoring.run(LoadFlow.find().getName(), LoadFlowParameters.load(), 4);
+            printHighAndLowVoltageConstraints(result).forEach(businessLogger::warn);
+            return Optional.of(result);
         } catch (Exception e) {
             businessLogger.error("Exception during voltage check : {}", e.getMessage());
             return Optional.empty();
         }
     }
 
+    protected List<String> printHighAndLowVoltageConstraints(final VoltageMonitoringResult result) {
```

Comment:
This function does not "print" anything. Maybe you could rename it "generateHighAndLow..."

Reply:
Taken into account in a new commit